lib/shoryuken.rb

```diff
@@ -95,6 +95,10 @@ def on_stop(&block)
       @stop_callback = block
     end
 
+    def config_rails_main_process
+      EnvironmentLoader.load_for_rails_main_process
+    end
```

Comment:
why expliciting calling it `rails`? For example, I'm not using Shoryuken with Rails, I ran it without Rails.